next/components/search/search-result.tsx

```diff
@@ -4,21 +4,23 @@ import { Badge } from "@/wunder-component-library/badge";
 
 export function SearchResult({ result, onClickLink }: ResultViewProps) {
   return (
-    <a
-      onClick={onClickLink}
-      rel="noopener noreferrer"
-      key={result.id.raw}
-      href={result.path.raw}
-      target="_blank"
-      className="my-4 block rounded bg-white p-8"
-    >
-      <h5 className="mb-4 text-xl font-bold text-primary-800">
-        {result.title.raw}
-      </h5>
-      <p className="mb-6 text-md text-scapaflow">{result.excerpt.raw}</p>
-      <Badge variant="info" size="sm">
-        {result.content_type.raw}
-      </Badge>
-    </a>
+    <li>
+      <a
+        onClick={onClickLink}
+        rel="noopener noreferrer"
+        key={result.id.raw}
```

Comment:
I guess this `key` can be removed if there are no complaints from React?

Reply:
Thanks for pointing this out! Actually as far as I know React wants there to be unique keys for list items especially when generated dynamically to keep track of them so I moved that to the <li> element.